docs/api.md

```diff
@@ -644,6 +644,33 @@ most.from([1, 2])
 	.observe(console.log.bind(console));
 ```
 
+### flatMapEnd
+
+####`stream.flatMapEnd(f) -> Stream`
+####`most.flatMapEnd(f, stream) -> Stream`
+
+Map the end event to a new stream, and begin emitting its values.  Note that `f` *must* return a stream.
```

Comment:
"Extend" is another pretty good word, but not sure how to incorporate it.  Any ideas?

Here's another angle with "replace".  Maybe with the ascii diagram, this is good enough:

"Replace the end signal with a new stream returned by `f`.  Note that `f` *must* return a stream."